solidity/contracts/interfaces/polygonZkevm/IBasePolygonZkEVMGlobalExitRoot.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: AGPL-3.0
```

Comment:
Can we use forge submodules or npm for polygon dependencies, don't prefer to have dependencies vendored here